### DISCUSSION TITLE
有更新日志吗？ 请问新增的ipv6选项是干嘛的

### DISCUSSION THREAD

**Discussion by Ecc2u** [vvbbnn00/WARP-Clash-API#102] [Q&A]
如题

**Comment by vvbbnn00** [comment]
打开ipv6后，返回的将都是ipv6的endpoint：https://github.com/vvbbnn00/WARP-Clash-API/pull/100

**Reply by kilvn** [reply]
能不能加个 v4v6 混合的选项，下拉也行  比如

v4
v6
混合

v6 资源太少了，比如我自己搭的代理服务 v6 才6个，v4 有 40 多个，如果选 v6 只有个位数是不太合适的
